alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/spring/integration/akka/AkkaConfigPropertyAdapter.java

```diff
@@ -58,7 +59,18 @@
               if (LOG.isTraceEnabled()) {
                 LOG.trace("Adding property {}={}" + key, value);
               }
+              
               propertyMap.put(key, value);
+              
+              if(name.startsWith(REDIS_NAME) && !propertyMap.containsKey(name)){
+                String readJournalKey = ("akka-persistence-redis.read-journal.redis.").concat(name.substring(REDIS_NAME.length()));
+                String journalKey = ("akka-persistence-redis.journal.redis.").concat(name.substring(REDIS_NAME.length()));
```

Comment:
why do we need to set up 3 configuration key with one value?

Reply:
> why do we need to set up 3 configuration key with one value?
It's OK, akka-persistence-redis contain three components, journal, snapshot, read-journal, their Redis configuration is the same.

https://github.com/safety-data/akka-persistence-redis/blob/master/src/main/resources/reference.conf